src/modules/lib/LibInterfaceId.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
What is the benefit of having a big file storing all the interfaceIDs? Isn't it easier to generate them on the fly since any contract checking them will know against what to compare? This way we avoid having to maintain a list, since we are going to be adding modules over time...

Reply:
My thought process was regarding the late dependency injection usecase. When creating a new contract, you might get an address that either you can accept as a particular type of module or ask additionally for it's interface ID in the metadata, which you can compare against all the interface IDs of the inverter network by making a call to this library. Without this library, you'll probably have to import a particular type of module into the module that receives the metadata to calculate and then compare the interface IDs.

If this sounds like gibberish, happy to delete the library as well.

Reply:
Hmm I see. Seems like overhead to me, what do you think, @FHieser ? Also, I think I'd remove the use in the Orchestrator, since it will only validate the core interface types.